test/language/line-terminators/S7.3_A3.4_T1.js

```diff
@@ -5,8 +5,7 @@
 info: Single line comments can not contain PARAGRAPH SEPARATOR (U+2029) inside
 es5id: 7.3_A3.4_T1
 description: Insert PARAGRAPH SEPARATOR (\u2029) into single line comment
-negative: ReferenceError
+negative: SyntaxError
 ---*/
 
-// CHECK#1
-eval("// single line \u2029 comment");
+// single line PS> ??? (invalid)
```

Comment:
> It's not easy to tell this line contains a PS character, but still looks better than using eval.

Yup. Refactoring like this isn't going to be pretty, but `eval` forces it to be a runtime error, which I'd rather avoid.

> Also, it seems the expected error should be the ReferenceError, which you can catch:

Since the behavior under test is a consequence of the grammar, I'd much rather express the test in a way that does not rely on runtime semantics.